src/test/functional/register-user-test.ts

```diff
@@ -0,0 +1,132 @@
+import {
+  assignRolesToParentRole,
+  createAssignableRoles,
+  createUserWithRoles
+} from './shared/testingSupportApi';
+
+import '../../main/utils/utils';
+
+Feature('Manage Existing User');
+import {config as testConfig} from '../config';
+import * as Assert from 'assert';
+import {randomData} from './shared/random-data';
+
+
+const PARENT_ROLE = randomData.getRandomRole();
+const ASSIGNABLE_CHILD_ROLE1 = randomData.getRandomRole();
+const ASSIGNABLE_CHILD_ROLE2 = randomData.getRandomRole();
+const DASHBOARD_USER_EMAIL = randomData.getRandomEmailAddress();
+
+BeforeSuite(async () => {
+  await createAssignableRoles(PARENT_ROLE);
+  await createAssignableRoles(ASSIGNABLE_CHILD_ROLE1);
+  await createAssignableRoles(ASSIGNABLE_CHILD_ROLE2);
+  // Assigning self role with the child role so the this user can also delete same level users
+  await assignRolesToParentRole(PARENT_ROLE, [ASSIGNABLE_CHILD_ROLE1, ASSIGNABLE_CHILD_ROLE2, PARENT_ROLE]);
+  await createUserWithRoles(DASHBOARD_USER_EMAIL, testConfig.PASSWORD, testConfig.USER_FIRSTNAME, [testConfig.RBAC.access, PARENT_ROLE]);
+});
+
+Scenario('I as a user should be able to register new user', async ({I}) => {
```

Comment:
```suggestion
Scenario('I as a user should be able to register new user', 
  {featureFlags: [BETA_ADD]},
  async ({I}) => {
```
I think you're missing the feature flag for this test

Reply:
Fixed in recent commit